src/store/index.js

```diff
@@ -173,10 +178,25 @@ export default createStore({
     },
   },
   actions: {
-    login({ commit }, token) {
-      const response = AuthService.login(token);
+    async login({ commit }, { token }) {
+      const { data, error } = await AuthService.login(token);
+
+      if (error) {
+        commit(SET_ALERT, {
+          success: false,
+          message: 'You are not authorized to access this website',
+        });
+        return;
+      }
 
-      commit(SET_USER, response);
+      commit(LOGIN, data.idToken);
```

Comment:
You're using `idToken` here and access token elsewhere, update the terminology and check the api response